src/util/utilization.jl

```diff
@@ -0,0 +1,126 @@
+
+"""
+    `run_UTIL(data::Dict{String,<:Any})`
+
+Returns a priority restoration order for damaged components in a single network.
+
+# Example
+```
+julia> run_UTIL(data)
+Dict{String, Any} with 6 entries:
+    "4" => [("branch", "2")]
+    "1" => [("gen", "1")]
+    "5" => [("branch", "3")]
+    "2" => [("gen", "2")]
+    "6" => [("gen", "3")]
+    "3" => [("branch", "1")]
+```
+"""
```

Comment:
This level of documentation detail may be a bit excessive given the maturity of this package.  I do agree the general API of restoration algorithms should be documented some place.  Maybe we should just add a special section to the docs markdown files where we talk about the API for all of these methods and then do an autodoc on each of the key heuristic functions?  Similar to how the PowerFlow docs are done in PowerModels?  https://lanl-ansi.github.io/PowerModels.jl/stable/power-flow/

Reply:
This makes sense.  In this function, I am specifically documenting the return format because it is not used anywhere else in the package.

Per our our discussion in December, I have added  a separate function for running util repair ordering and solving a power flow.  This function is called `run_utilization` and is documented similar to the examples in PowerModels.

This function is now called `utilization_repair_order`.  

As the future heuristic functions are added, I will create the autodocs for them.